u4a-component/charts/cluster-component/values.yaml

```diff
@@ -14,7 +14,7 @@ ingress-nginx:
     ingressClass: portal-ingress
     image:
       repository: hub.tenxcloud.com/u4a-component/ingress-nginx-controller
-      tag: "0.30.0"
+      tag: "v1.3.0"
```

Comment:
`0.30.0` is this the community version ?

Reply:
> `0.30.0` is this the community version ? 

@nkwangleiGIT Any suggestions?

Reply:
It seems this version naming style has been changed.
See https://github.com/kubernetes/ingress-nginx#supported-versions-table

![image](https://github.com/kubebb/building-base/assets/30621793/0f8885c4-10cc-414c-9060-e07c53280f0d)

Reply:
`v1beta1 ` got deprecated since v1.22, we can use v1.3.0 as the default nginx version. 
@0xff-dev sync building base with our installer.